src/main/java/org/jabref/logic/formatter/casechanger/CamelFormatter.java

```diff
@@ -0,0 +1,42 @@
+package org.jabref.logic.formatter.casechanger;
+
+import java.util.stream.Collectors;
+
+import org.jabref.logic.cleanup.Formatter;
+import org.jabref.logic.l10n.Localization;
+
+public class CamelFormatter extends Formatter {
+
+    @Override
+    public String getName() {
+        return Localization.lang("Camel");
+    }
+
+    @Override
+    public String getKey() {
+        return "camel";
```

Comment:
Did you look at the other formatters for inspriation?

```suggestion
        return "Camel case";
```